modules/TestUtils.js

```diff
@@ -28,6 +28,12 @@ export const isArray = (object) =>
   Array.isArray(object)
 
 /**
+ * Returns true if the given object is a set.
+ */
+export const isSet = (object) =>
+  Object.prototype.toString.call(object) === '[object Set]'
```

Comment:
this is fakeable since ES6 has `Symbol.toStringTag` - `Object#toString` output is not reliable >= ES6